core/primer.md

```diff
@@ -674,3 +674,64 @@ in the entity in order to be model compliant.
 
 Likewise, implementations of the server must validate the entire entity
 against the new model, not just a subset of the entity's attributes.
+
+# Why are `ancestor` attributes that point to themselves allowed?
+
+The [`ancestor` attribute](spec.md#ancestor-attribute) is used to build a
+hierarchy of Versions to facilitate compatibility checking when the
+[`compatibility` attribute](spec.md#compatibility-attribute) is set.
+Generally speaking, implementations are not expected to allow users to
+explicitly create multiple roots for a single Version. However, the
+specification does not mandate this as there may be cases where it is useful
+to allow for multiple roots. In some cases it may even be unavoidable, for
+example when the `maxversions` attribute (See `groups.resources.maxversions`
+under [Resource Model](spec.md#registry-model)) is set to a value that forces
+pruning of the Version tree. In such cases, when deleting the oldest version,
+this could result in a new root being created when there are multiple
+decedents of the deleted Version.
+
+# Creating versions with `singleversionroot` and a `maxversions` limit
+
+When the `groups.resources.maxversions` attribute of the [Resource
+Model](spec.md#registry-model) is set to a specified value for a Resource, the
+server will prune versions when attempting to create a new version that would
+exceed this value. However, there may be cases in which the creation of new
+versions may be blocked, due to server being unable to prune versions.
+This may occur when the `groups.resources.singleversionroot` attribute of
+the [Resource Model](spec.md#registry-model) is set to `true`.
+
+Consider a scenario in which 3 versions exist: v1 is the root (and therefore
```

Comment:
v->V above
and you can just say `v1` instead of "its versionid" in the next line. To be extra clear.